src/display/api.js

```diff
@@ -513,6 +513,12 @@ async function _fetchDocument(worker, source, pdfDataRangeTransport, docId) {
     }
   );
 
+  // Release the TypedArray data, when it exists, since it's no longer needed
+  // on the main-thread *after* it's been sent to the worker-thread.
+  if (source.data) {
+    source.data = null;
+  }
+
```

Comment:
Could it make sense to transfer the data in the worker ?

Reply:
I though about that, but decided against doing it since that may end up breaking someone's code. Given that can't know how third-party users are invoking `getDocument`, it's thus possible that a user want to do *something* with the TypedArray after calling `getDocument` and it'd be both surprising (and a breaking API-change) if we just transferred this data.

(And note that none of this applies to the Firefox PDF viewer either.)